onnxruntime/server/main.cc

```diff
@@ -52,25 +56,16 @@ int main(int argc, char* argv[]) {
     exit(EXIT_FAILURE);
   }
 
-  const auto env = std::make_shared<server::ServerEnvironment>(config.logging_level);
+  const auto env = std::make_shared<server::ServerEnvironment>(config.logging_level, std::make_shared<spdlog::sinks::stdout_sink_st>());
```

Comment:
We need to have both stdout sink and syslog sink for spdlog. Otherwise, the log will not be captured by AzureML logger in the container.